### ISSUE TITLE
Hardcoded architecture in apt list file?

### ISSUE THREAD

**Issue by Gregger83** [Icinga/docker-icinga2#37]
Hi all,

is there a special reason, why you hard-coded the architecture in file [action-base.list](https://github.com/Icinga/docker-icinga2/blob/master/action-base.list)

In other repos I found some kind of 

`echo "deb [arch=$(dpkg --print-architecture)] https://download.docker.com/linux/debian buster stable" > /etc/apt/sources.list.d/repo.list`

Regards.

**Comment by Al2Klimov** [comment]
Hello @Gregger83 and thank you for reporting!

Do you have any problems w/ non-amd64 due to this?

Best,
AK

**Comment by Gregger83** [comment]
Hi @Al2Klimov ,

I tried to figure out, why I get the same error (as mentioned in #27) on my RaspberryPi 3.

`$uname -a`
`Linux raspberrypi 5.4.72-v7+ #1356 SMP Thu Oct 22 13:56:54 BST 2020 armv7l GNU/Linux`

Otherwise, when every base image used by this image, supports arm architecture, why hardcoding a particular architecture in an apt repo?

I tried to build my own image by temporary hardcoding 🤣 armhf instead of *64 to check, if that makes any difference but actually I am struggling with a "COPY icinga2-bin" error.

Is there any HowTo that explains, how I can build that image on my system?

Thanks in advance.

Regards.
